src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -574,6 +589,9 @@ private void NodePortContextMenu(object obj)
 
         private bool CanAutoComplete(object parameter)
         {
+            MouseButtonEventArgs evArgs = parameter as MouseButtonEventArgs;
```

Comment:
Actually this should go in private bool AutoComplete(object parameter) ; CanAutoComplete can be called by the runtime in other contexts.